app/api/migrations/0008_auto_20210107_2108.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.13 on 2021-01-07 21:08
```

Comment:
merge all migrations when finished with this feature. you could also name the migration, so it's not called "auto something"